deptry/virtualenv_finder.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import os
+import sys
+from itertools import chain
+from pathlib import Path
+from typing import Optional
+
+from deptry.compat import metadata
+
+
+def find_site_packages_in(root: Path) -> Optional[Path]:
+    """Find site packages directory under a virtual environment root
+
+    Two layouts are tried: `lib/pythonX.X/site-packages` and
+    `Lib/site-packages`, the latter specific to Windows.
+    """
+    search = chain(root.rglob("lib/python*/site-packages"), root.rglob("Lib/site-packages"))
+    try:
+        return next(search)
+    except StopIteration:
+        return None
+
+
+def running_in_project_virtualenv(
+    project_name: str, *, prefix: str, base_prefix: str, active_virtual_env: Optional[str] = None
+) -> bool:
+    """Determine if executed by the interpreter in the project's virtual environment
+
+    If we are executed from virtual environment, the `prefix` argument
+    will be set to the virtual environment's directory, whereas
+    `base_prefix` will point to the global Python installation
+    used to create the virtual environment.
+
+    The `active_virtual_env` arguments holds the value of the
+    VIRTUAL_ENV environment variable, that tells with good reliability
+    that a virtual environment has been activated in the current shell.
+    """
+
+    # Gobal installation
+    if prefix == base_prefix:
+        return False
+
+    # No virtualenv has been activated. Unless the project name is in
+    # the interpreter path, assume we are not in the project's virtualenv
+    if not active_virtual_env:
+        return project_name in prefix
+
+    # A virtualenv has been activated. But if `deptry` was installed gloabally using
+    # `pipx`, we could be running in another installation.
+    return active_virtual_env == prefix
+
+
+def guess_virtualenv_site_packages(project_root: Path, active_virtual_env: Optional[str] = None) -> Optional[Path]:
+    """Try to locate a project's virtualenv packages using environment and popular locations"""
+    site_packages = None
+    possible_roots = [
+        Path("~/.virtualenvs").expanduser() / project_root.name,
+        project_root / ".venv",
```

Comment:
```suggestion
        project_root / ".venv",
        project_root / "venv",
```

Which is also a common location; https://docs.python.org/3/library/venv.html